packages/storybook/stories/va-radio-uswds.stories.jsx

```diff
@@ -0,0 +1,325 @@
+import React, { useState, useEffect } from 'react';
+import { VaRadio, VaRadioOption} from '@department-of-veterans-affairs/web-components/react-bindings';
+
+import {
+  getWebComponentDocs,
+  componentStructure,
+  propStructure,
+  StoryDocs,
+} from './wc-helpers';
+
+VaRadio.displayName = 'VaRadio';
+VaRadioOption.displayName = 'VaRadioOption';
+
+const radioDocs = getWebComponentDocs('va-radio');
+const radioItem = getWebComponentDocs('va-radio-option');
+
+export default {
+    title: `USWDS/Radio button USWDS`,
+    id: 'uswds/va-radio',
+  subcomponents: componentStructure(radioItem),
+  parameters: {
+    componentSubtitle: `va-radio web component`,
+    docs: {
+      page: () => <StoryDocs data={radioDocs} />,
+    },
+  },
+};
+
+const vaRadioConst = args => {
+  const {   
+    'enable-analytics': enableAnalytics,
+    error,
+    label,
+    hint,
+    required,
+    tile,
+    uswds,
+    ...rest
+  } = args;
+
+  return (
+    <va-radio
+        enable-analytics={enableAnalytics}
+        error={error}
+        label={label}
+        required={required}
+        uswds={uswds}
+        hint={hint}
+    >
+    <va-radio-option 
+      id="soujourner-truth" 
+      label="Soujourner Truth" 
+      name="group1" 
+      value="1" 
+      uswds={uswds} 
+    />
+    <va-radio-option
+      id="frederick-douglass"
+      label="Frederick Douglass"
+      name="group1"
+      value="2"
+      uswds={uswds}
+    />
+    <va-radio-option
+      id="booker-t-washington"
+      label="Booker T. Washington"
+      name="group1"
+      value="3"
+      uswds={uswds}
+    />
+    <va-radio-option
+      id="george-washington-carver"
+      label="George Washington Carver"
+      name="group1"
+      value="4"
+      disabled
+      uswds={uswds}
+    />
+  </va-radio>
+  )
+}
+
+const Template = args => vaRadioConst(args);
+
+const I18nTemplate = args => {
+  const [lang, setLang] = useState('en');
+
+  useEffect(() => {
+    document.querySelector('main').setAttribute('lang', lang);
+  }, [lang]);
+
+  return (
+    <div>
+      <button style={{fontSize: '1rem'}} onClick={e => setLang('es')}>Español</button>
+      <button style={{fontSize: '1rem'}} onClick={e => setLang('en')}>English</button>
+      <button style={{fontSize: '1rem'}} onClick={e => setLang('tl')}>Tagalog</button>
+      <br /><br />
+      {vaRadioConst(args)}
+    </div>
+)};
+
+const ReactBindingExample = ({
+  'enable-analytics': enableAnalytics,
+  error,
+  label,
+  required,
+}) => {
+  return (
+    <>
+    <VaRadio 
+        uswds
```

Comment:
There might be a way to make this a default but I wanted the `uswds` prop to display in the examples on Storybook and I didn't think it would display any other way. But I'm not a Storybook expert so if you see a way to set this as a default while still showing in the example, I can definitely update it.

![Screen Shot 2022-12-01 at 3 44 37 PM](https://user-images.githubusercontent.com/872479/205165436-aa2aa798-0d1b-48ac-9760-6cb9391688ec.png)